packages/ui/cypress/support/page-objects/components/accountDisplay.ts

```diff
@@ -0,0 +1,7 @@
+export const accountDisplay = {
```

Comment:
Decided to create this a test object itself as it appears all over the app and then when checking for it's presence we can get cypress to yield the parent object and then check 'within' it to make sure we are checking the right ones (as account display details could appear on a page in multiple places etc)